app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,40 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        =bootstrap_form_for [:admin, @video], html: {class: "form-horizontal"} do |f|
+          %ul.nav.nav-tabs
+            %li
+              %a(href="/ui/admin_views_payments") Recent Payments
+            %li.active
+              %a(href="") Add a New Video
+          %br
+          %fieldset
+            .form-group
+              .control-label.col-sm-3= f.label :title, 'Title'
+              .col-sm-3
+                = f.text_field :title, hide_label: true
```

Comment:
OK, thanks for the homework, doing it right the way~~ ^^
I indeed not familiar with the rails form enough~